runtime/codert_vm/arm64nathelp.m4

```diff
@@ -321,8 +321,10 @@ OLD_DUAL_MODE_HELPER_NO_RETURN_VALUE(jitPutFlattenableStaticField,3)
 OLD_DUAL_MODE_HELPER(jitLoadFlattenableArrayElement,2)
 OLD_DUAL_MODE_HELPER_NO_RETURN_VALUE(jitStoreFlattenableArrayElement,3)
 SLOW_PATH_ONLY_HELPER_NO_RETURN_VALUE(jitResolveFlattenableField,3)
-FAST_PATH_ONLY_HELPER(jitLookupDynamicInterfaceMethod,3)
+
+ifdef({ASM_J9VM_OPT_OPENJDK_METHODHANDLE},{
```

Comment:
I believe this should be in the `METHOD_INVOCATION` section.

Reply:
Ah, ok, I'll move this one and check/move the corresponding definitions in the other nathelp.m4 files

Reply:
Hm, now I'm not so sure. It looks to me like the `METHOD_INVOCATION` section is trying to skip preserving registers that the JIT-compiled code expects to be clobbered by the call anyway. But this helper is a separate tree in the IL, so we'll call the helper first and then set up private linkage for the call afterward. The JIT could try to keep a value across the helper call in a register that wouldn't be preserved by a `METHOD_INVOCATION` helper. It's unfortunate that this way we'll end up preserving registers that are about to be clobbered by the real call, but to avoid that we'd have to fuse the two calls together

Reply:
OK, we'll leave it as-is.